lisa/analysis/rta.py

```diff
@@ -0,0 +1,243 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
IIRC you had issues with making this live in `lisa.wlgen.rta.py`, which is why it's in `lisa.analysis`. However, the analysis module has this fancy proxy. Doesn't that mean we'll get a `trace.analysis.rta` member when creating `Trace` objects?

Reply:
Currently no, the proxy only gives access to subclasses of AnalysisBase. PerfAnalysis does not inherit from it, since it does not use a trace at all, and is using a different set of data. Therefore it would not make sense to make it available as an attribute of lisa.Trace instances.